lightning/src/ln/channelmanager.rs

```diff
@@ -4723,7 +4723,7 @@ impl<Signer: Sign, M: Deref, T: Deref, K: Deref, F: Deref, L: Deref> ChannelMana
 					if chan_entry.get().get_counterparty_node_id() != *counterparty_node_id {
 						return Err(MsgHandleErrInternal::send_err_msg_no_close("Got a message for a channel from the wrong node!".to_owned(), msg.channel_id));
 					}
-					let (closing_signed, tx) = try_chan_entry!(self, chan_entry.get_mut().closing_signed(&self.fee_estimator, &msg), channel_state, chan_entry);
+					let (closing_signed, tx) = try_chan_entry!(self, chan_entry.get_mut().closing_signed(&&self.fee_estimator, &msg), channel_state, chan_entry);
```

Comment:
Hmm yeah I think I can now with how FeeEstimator currently looks.

Reply:
So yeah we don't need it in _this_ case, but if we make `ChannelManager::fee_estimator` a `LowerBoundedFeeEstimator`, then it seems necessary as we're working with something that's not a Deref to a FeeEstimator.

Reply:
I think the main issue here is that the signature expects `&F` for first argument and F: Deref where F::Target: FeeEstimator.

So it's a ref to a deref to a FeeEstimator, so that's a bit of a problem here :/
Errors with just a single `&` in the case where ChannelManager::fee_estimator is a LowerBoundedFeeEstimator, because of the signature.